Gifski/ShowBehavior.swift

```diff
@@ -0,0 +1,38 @@
+struct ShowBehavior {
```

Comment:
I think this type should be nested in `Tooltip`. It's not really useful on it's own.